serverlessworkflow/modules/ROOT/pages/integrations/serverless-dashboard-with-runtime-data.adoc

```diff
@@ -1 +1,315 @@
-//= Displaying workflow runtime data in serverless dashboards
\ No newline at end of file
+= Displaying workflow runtime data in serverless dashboards
+
+This document describes steps to create serverless dashboards to monitor Kogito Serverless Workflows. 
+
+To build dashboards we need data and it can be easily acquired from Kogito Serverless Workflow using any of the following approaches:
+
+* Enabling **Metrics**
+* Integrating with **Data Index** to consume data using its REST API
+* Using custom mechanisms (such as `ProcessInstanceListener`)
+
+In this document we will explore creating serverless dashboards for metrics and for Data Index GraphQL REST API.
+
+== Serverless Dashboards with Dashbuilder
+
+Serverless Dashboards are visualizations that run entirely on the client side. We will use https://www.dashbuilder.org[Dashbuilder] to create serverless dashboards: it consumes data from any JSON source and the dashboard is defined in YML format.
+
+To use Dashbuilder you just have to uncompress the `dashbuilder-client` NPM package and edit the file *setup.js* to link your YML files.
```

Comment:
Can you elaborate a bit more? So it's just unzip, modify the js and run npm install? Do we have a doc in the dashbuilder website that teaches how to build an image for it? Maybe it would be good to link it here, so that users can push to a cluster with the application.

One question, though: what's the recommendation? Have one dashbuilder per application or one to rule them all? 😆

Reply:
I am adding a link  to the dashbuilder docs for now - I hope to later edit this part to include a link to our official online client. For now the dashbuilder client installation is the only official way to run YML files!

About the number of dashboards, you can use as many as you want :-D it should not impact performance - the only difference is the home page, which will have a list of dashboards.